assets/src/application/ui/forms/espressoForm/renderers/FormRenderer.tsx

```diff
@@ -29,21 +29,23 @@ const FormRenderer: React.FC<FormRendererProps> = (props) => {
 	} = props;
 
 	const formOutput = (
-		<div className='form-wrapper'>
-			<Form layout={layout}>
-				{sections.length ? <RenderSections sections={sections} /> : null}
+		<div className='ee-form'>
```

Comment:
nice and simple;
with my commit I was thinking that the container div should be in the `EspressoForm`